### PR TITLE
Reapplied the helper generator

### PR DESCRIPTION
**Pull request by pasaperez-work**
Pull-request for issue https://github.com/slingr-stack/platform/issues/13746 created by @pasaperez-slingr

## What it does?

The helper generator has been reapplied with all the latest updates.
Changed the flow steps icons.
Cleaned the custom Java code to make it easier to migrate to the service.

## How to test it?

Review helpers.

## Implementation notes

There are no implementation notes

## Deployment notes

There are no deployment notes

### CODE REVIEW

**Comment by sebasmoyano** [comment]
#approve

File: src/main/resources/endpoint.properties
Comment (agreggio-slingr) [inline src/main/resources/endpoint.properties:1]: It should not be modified the endpoint.properties file to run locally. Instead, something like endpoint_local.properties should be created.

File: flowSteps/apiCallPandadoc/icon.png
Comment (agreggio-slingr) [inline flowSteps/apiCallPandadoc/icon.png:1]: If we are going to use the endpoint icon for each step, I would consider the possibility of using the one from the root directory and removing it from the steps. This way, we would avoid duplicating files and if the icon changes, it changes in all the steps.

File: src/main/resources/endpoint.properties
Comment (pasaperez-work) [reply src/main/resources/endpoint.properties:1]: I agree, now you only have added the missing properties with default values.

File: flowSteps/apiCallPandadoc/icon.png
Comment (pasaperez-work) [reply flowSteps/apiCallPandadoc/icon.png:1]: The same icon is used, but it is not the same file, the endpoint file is 48x48 and the steps file has been rescaled to 18x18. 
Even so the step must have by definition the png, the js and the json.

**Pull request merged**